src/Commands/CreateOrEditTaskParser.ts

```diff
@@ -27,13 +27,18 @@ export const taskFromLine = ({ line, path }: { line: string; path: string }): Ta
         DateFallback.fromPath(path), // set the scheduled date from the filename, so it can be displayed in the dialog
     );
 
+    const { setCreatedDate } = getSettings();
+    const createdDate = setCreatedDate ? window.moment().startOf('d') : null;
```

Comment:
What was the reason for adding `.startOf('d')` please?

Reply:
I had an issue with `moment` comparison in the tests due to time difference, so I added this. But if I try to remove it now, the tests still run fine so I conclude this is useless. Maybe I can include this in the additional PR?

Reply:
Thanks.